pelux.xml

```diff
@@ -71,14 +71,14 @@
 
   <project remote="code.qt"
            upstream="sumo"
-           revision="880ecb7a6e993a986d24c9a8b813bf2a48f51d24"
+           revision="75e93a6ecb121d9aa0ee4aa986b3eef0dd20ea22"
```

Comment:
May make sense to use QtAS-5.12.1, but meta-qt5 shall also be bumped with same tag